cli/src/helpers/installDependencies.ts

```diff
@@ -9,7 +9,21 @@ export const installDependencies = async (projectDir: string) => {
   const pkgManager = getUserPkgManager();
   const spinner = ora(`Running ${pkgManager} install...\n`).start();
 
-  await execa(pkgManager, ["install"], { cwd: projectDir });
+  // FIXME: temp fix for next-auth with node 18
+  // see: https://github.com/nextauthjs/next-auth/issues/4575
+  if (process.versions.node.startsWith("18")) {
```

Comment:
```suggestion
  if (
    process.versions.node.startsWith("18") ||
    process.versions.node.startsWith("19")
  ) {
```

Reply:
I'm guessing the same goes for 19 so we should check that too since that's now a thing